### PR TITLE
Hg 669 Fixing article comments query param regression

### PR DESCRIPTION
**Pull request by lizlux**
This is a bit of a hacky way to get the article comments query param working again. It really seems to belong on the article controller, not the application controller. 

I'm submitting this PR in the interest of time and getting it done within our sprint. I'll continue to look for a better solution though. 

@rogatty @kenkouot @katanka

### CODE REVIEW

**Comment by rogatty** [comment]
Please create a follow up ticket and add comments to the code with its name.

LGTM

**Comment by lizlux** [comment]
@rogatty https://wikia-inc.atlassian.net/browse/HG-815

File: front/scripts/main/controllers/ApplicationController.ts [hunk 1]
```diff
@@ -7,12 +7,15 @@ App.ApplicationController = Em.Controller.extend(App.LoadingSpinnerMixin, App.Al
 	// This has to be here because we need to access media from ArticleController model to open lightbox
 	// TODO: Should be refactored when decoupling article from application
 	needs: ['article'],
-	queryParams: ['file', 'map', {
-		noAds: 'noads'
-	}],
+	queryParams: ['file', 'map',
+		{ noAds: 'noads' },
```
Comment (kenkouot) [inline front/scripts/main/controllers/ApplicationController.ts]: no whitespace inside {}

**Comment by kenkouot** [comment]
LGTM, thank you for making the follow-up ticket.

**Pull request merged**